### ISSUE TITLE
Using UART0 pins as Output

### ISSUE THREAD

**Issue by Fareed99** [atomic14/esp32-s3-pinouts#23]
As mentioned in your repo, the GPIO43 and GPIO44 are connected to UART0 by default.
But if we're using the USB for uploading code and for the serial console, can we use the 43 and 44 GPIOs as output pins? Let's say for controlling a Nexopixel ring.

**Comment by cgreening** [comment]
Yes — when you upload and run the console over the **native USB** port, UART0 is free and GPIO43/44 can be used as ordinary GPIOs (e.g. driving a NeoPixel ring). Just mind the boot-time pull-ups, and disable/redirect the console (`ESP_CONSOLE_NONE`) if early ROM/bootloader logging on those pins is a problem. Documented in 568a2c2.
